libs/utils/src/decorators/is-int-value.decorator.ts

```diff
@@ -19,7 +19,7 @@ export function IsIntValue(validationOptions?: IsIntValueValidationOption) {
 
           if ((typeof value === 'string' && re.test(value)) || typeof value === 'number') {
             const numberValue = BigInt(value);
-            if (validationOptions.minValue !== undefined) {
+            if (validationOptions.maxValue !== undefined) {
```

Comment:
there is an already existing IsInt() decorator that support that usecase. Our usecase was more limiting the range of the int mainly on the min side and sometimes on both sides. But I can make both of them optional anyway.

Reply:
That might be best; it's confusing to have to use one decorator if you want a `minValue`, and another decorator if you don't.

Also, it might be useful to accept another option, `accept_strings: boolean` that controls whether the value must be an actual number, or if it can also be a string representation of a number. The `class-validator` package has `@IsNumber()` and `@IsNumberString()`, both of which take some options worth examining.

Actually, the more I think about it, in the interest of velocity & small PRs, let's go ahead with this and consider enhancing our stable of decorators in a future PR.